lib/streamlit/script_runner.py

```diff
@@ -206,14 +202,29 @@ def _get_script_run_ctx(self) -> ScriptRunContext:
             )
         return ctx
 
-    def _process_request_queue(self) -> None:
-        """Process the ScriptRequestQueue and then exits.
+    def _run_script_thread(self) -> None:
+        """The entry point for the script thread.
```

Comment:
`_process_request_queue` is now called `_run_script_thread`, because it is the main entry point for the script thread. (It does literally "process the request queue", but it also runs the ScriptRunner's script and handles cleanup when the thread is shutting down.)